app/uk/gov/hmrc/soletraderidentificationfrontend/models/SoleTraderDetailsMatching.scala

```diff
@@ -74,7 +70,6 @@ object SoleTraderDetailsMatching {
           case NinoNotDeclaredButFoundKey => JsSuccess(NinoNotDeclaredButFound)
           case NinoNotFoundKey => JsSuccess(NinoNotFound)
           case DeceasedCitizensDetailsKey => JsSuccess(DeceasedCitizensDetails)
-          case NinoIsFraudulentKey => JsSuccess(NinoIsFraudulent)
```

Comment:
There should not be fraudulent ninos in mongodb? Right!! Otherwise we have to leave it and remove it in another ticket.

Reply:
No, there won't be. If there is then a user might get a tech difficulties page but they're using a fraudulent nino so I think it's okay. They would just need to start from the calling service new again